docs/docsite/rst/scenario_guides/kubernetes_scenarios/k8s_inventory.rst

```diff
@@ -0,0 +1,95 @@
+.. _k8s_ansible_inventory:
+
+*****************************************
+Using Kubernetes dynamic inventory plugin
+*****************************************
+
+.. contents:: Topics
+
+Kubernetes Dynamic Inventory Plugin
+===================================
+
+
+The best way to interact with your Pods is to use the Kubernetes dynamic inventory plugin, which dynamically queries Kubernetes APIs using ``kubectl`` command line available on controller node and
+tells Ansible what Pods can be managed.
```

Comment:
```suggestion
```

Reply:
Translation has problems with sentences being split up on different lines like this was.

Reply:
Got it. Thanks for letting me know.